public/js/action-list.js

```diff
@@ -82,14 +82,26 @@
                 $list.find('.list-item.active').removeClass('active');
                 $item.addClass('active');
             }
+
+            // For items that do not have a bottom status bar like Downtimes, Comments...
+            if (! $list.closest('.container').find('.footer').length) {
+                $list.closest('.container').append('<div class="footer automatically-added"></div>');
+            }
         } else {
             $list.find('.list-item.active').removeClass('active');
             $item.addClass('active');
         }
 
         $activeItems = $list.find('.list-item.active');
+        var bottomStatusBar = $('.footer');
```

Comment:
This way you not only find a `.footer` in the current container, but also in the menu and other containers. Selecting elements with such a broad rule isn't good practice. Preferably, do it the same as on line 87. You may also consider now to cache the result of `$list.closest('.container')` :wink: